router/router_test.go

```diff
@@ -2240,6 +2240,34 @@ var _ = Describe("Router", func() {
 			})
 		})
 	})
+
+	FDescribe("cert selection", func() {
```

Comment:
I think the "Focus" has be removed

Reply:
Uhh sorry, didn't notice that. I'll fix it :)

Reply:
see #409